test/E2ETests/E2EApps/E2EApp/E2EApp.csproj

```diff
@@ -42,8 +43,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.WebUtilities" Version="2.2.0" />
-    <PackageReference Condition="$(TestBuild) == 'true'" Include="Microsoft.Azure.Functions.Worker" Version="1.12.1-preview1" />
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.9.0-preview1" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.13.0-preview1-local202305031031" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.10.0-local202305031056" />
```

Comment:
This change shouldn't be checked in; it's automated by the e2e setup script right?

Reply:
for the worker.sdk package I needed to specify @surgupta-msft 's bypass deferred binding changes. Is there a way to get around this?

Reply:
Will update this once packages have been released on public feeds